pinot-common/src/main/java/com/linkedin/pinot/common/config/AbstractTableConfig.java

```diff
@@ -70,8 +71,13 @@ public static AbstractTableConfig init(String jsonString) throws JSONException,
         loadIndexingConfig(new ObjectMapper().readTree(tableJson.getJSONObject("tableIndexConfig").toString()));
     QuotaConfig quotaConfig = null;
     if (tableJson.has(QuotaConfig.QUOTA_SECTION_NAME)) {
-      quotaConfig = loadQuotaConfig(new ObjectMapper().readTree(
-          tableJson.getJSONObject(QuotaConfig.QUOTA_SECTION_NAME).toString()));
+      try {
+        quotaConfig = loadQuotaConfig(
+            new ObjectMapper().readTree(tableJson.getJSONObject(QuotaConfig.QUOTA_SECTION_NAME).toString()));
+      } catch(ConfigurationRuntimeException e) {
+        LOGGER.error("Invalid quota configuration value for table: {}", tableName);
+        throw e;
```

Comment:
As we discussed, we need to throw here because the table config is bad.

Added a test and verified usages